pytorch/inference/docker/1.12/py3/Dockerfile.cpu

```diff
@@ -50,7 +55,7 @@ ARG PT_TORCHVISION_URL
 ARG PT_TORCHAUDIO_URL
 ARG PT_TORCHDATA_URL
 
-ARG TS_VERSION=0.6.0
+ARG TS_VERSION
```

Comment:
Why do we redefine these args with no values at various points in teh file?

Reply:
To my knowledge it s got to do with scoping. The requirement here is to declare it at one place and re-use it across each build stage.

Reply:
Do we have any way of verifying that TS version gets added as expected in the EC2 images?

Reply:
comment below